### DISCUSSION TITLE
withdrew RenBTC from solana to ETH but only got Wormhole Wrapped Token, How do I send it back?

### DISCUSSION THREAD

**Discussion by misterbaka** [wormhole-foundation/wormhole#246] [Problems / bugs] [answered]
Hi,

I bought some renbtc on solana and used wormhole to send it over to ETH.  I didnt expect to get a wormhole wrapped token which is useless on ETH.  I want to send it back but I dont see a way to bring up that token on the wormhole UI.

Can someone please help?

Thanks!

**Comment by hendrikhofstadt** [comment, marked as answer]
Please contact the operator of the UI you used. We don’t provide a UI or support for it.

**Comment by leoluk** [comment]
You can create issues for wormholebridge.com here: https://github.com/solana-labs/oyster/issues

**Comment by leoluk** [comment]
In case you haven't seen - wormholebridge.com recently added a bunch of new wrapped tokens, so you may be able to transfer them back. If not, make sure to contact them to add this pair!
